ts/features/services/home/components/FeaturedInstitutionCard.tsx

```diff
@@ -0,0 +1,125 @@
+import {
+  Avatar,
+  H6,
+  IOColors,
+  IOVisualCostants,
+  VSpacer
+} from "@pagopa/io-app-design-system";
+import { OrganizationFiscalCode } from "@pagopa/ts-commons/lib/strings";
+import React from "react";
+import { Dimensions, StyleSheet, View } from "react-native";
+import Placeholder from "rn-placeholder";
+import { WithTestID } from "../../../../types/WithTestID";
+import { logoForInstitution } from "../utils";
+
+export type FeaturedInstitutionCardProps = WithTestID<{
+  id: string;
+  name: string;
+  accessibilityLabel?: string;
+  isNew?: boolean;
+  onPress?: (id: string) => void;
+}>;
+
+export const CARD_WIDTH =
+  Dimensions.get("window").width - IOVisualCostants.appMarginDefault * 2;
+
+const styles = StyleSheet.create({
+  cardBaseContainer: {
+    height: 72,
+    width: CARD_WIDTH,
+    paddingHorizontal: 12,
+    borderRadius: 8,
```

Comment:
And replace the hardcoded values with previously defined attributes. Also enable smooth corners on iOS only (will be visible when we upgrade to `0.71`) 
```suggestion
    paddingHorizontal: cardPaddingHorizontal,
    borderRadius: cardBorderRadius,
    borderCurve: "continuous",
```

Reply:
aa1955420

Reply:
@giuseppedipinto Remember to add the `borderCurve` property as well

Reply:
@dmnplb 2fb32f2f3